src/components/footer/Footer1.jsx

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import './footer1.css';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import YouTubeIcon from '@mui/icons-material/YouTube';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import mobile from '../assets/mobile.png';
+import returnOrder from '../assets/return.png';
+import original from '../assets/original.png';
+
+export default function Footer1() {
+  const year = new Date().getFullYear();
+  return (
+    <div className="footer">
+      <div className="footer_first">
+        <div className="footer_first_one">
+          <h5 className="footer_heading">Online Shopping</h5>
```

Comment:
In DOM Contents like "Online Shopping" can be fetched via some data model or constants or init states and will be followed for rest of the classes